### ISSUE TITLE
Convert from Flat df back to xml - is it possible

### ISSUE THREAD

**Issue by scherbakovdmitri** [jsugarelli/flatxml#4]
Hello, nice package! Is it possible to save back to xml after I've done manipulations on flat df? 
thanks!

**Comment by jsugarelli** [comment]
No, unfortuantely, this is not possible at the moment. Conversion is one-way, if you like. But this shouldn't be too difficult to develop, so it might be nice feature for a future version.
